### PR TITLE
Cut Release 0.5.0-rc1 for torchserve integration test

### PR DESCRIPTION
**Pull request by yuzisun**
**Special notes for your reviewer**:

1. Cut 0.5.0-rc1 release for testing torchserve integration https://github.com/kubeflow/kfserving/pull/1161
2. Merging logger to agent image https://github.com/kubeflow/kfserving/pull/1223
3. Kafka event source fixes https://github.com/kubeflow/kfserving/pull/1238

**Release note**:

```release-note

```

### CODE REVIEW

File: config/configmap/inferenceservice.yaml [hunk 1]
```diff
@@ -62,8 +62,8 @@ data:
         "pytorch": {
```
Comment (chauhang) [inline config/configmap/inferenceservice.yaml:62]: @yuzisun For TorchServe, do we need the v2 entries under "pytorch" as well?

**Review by chauhang — COMMENTED** [review]
Please see comment inline and provide clarification if entry for TorchServe needs to be added here as well

**Comment by yuzisun** [comment]
> Please see comment inline and provide clarification if entry for TorchServe needs to be added here as well

@chauhang it is already added [here](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml#L74) by @jagadeeshi2i , the generated release should already include it.

**Review by yuzliu — APPROVED** [review]

**Pull request merged**